plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryFilterQueryBuilder.java

```diff
@@ -145,10 +148,7 @@ else if (singleValues.size() > 1) {
 
     private Optional<String> toPredicate(String columnName, String operator, Object value, BigQueryColumnHandle column)
     {
-        Optional<String> valueAsString = convertToString(column.getTrinoType(), column.getBigqueryType(), value);
-        if (valueAsString.isEmpty()) {
-            return Optional.empty();
-        }
-        return Optional.of(quote(columnName) + " " + operator + " " + valueAsString.get());
+        return convertToString(column.getTrinoType(), column.getBigqueryType(), value)
```

Comment:
Revert. This variable is helpful when debugging the code.

Reply:
Yeah, not mixing things up sounds reasonable 👍 I'll update